src/components/Elements/MenuItem.js

```diff
@@ -4,10 +4,17 @@ import classNames from 'classnames';
 import { Icon } from 'network-canvas-ui';
 
 /**
-  * Renders a menu item. Image is based on interfaceType.
+  * Renders a menu item. Image is based on icon.
   */
 function MenuItem(props) {
-  const { onClick, interfaceType, isActive, menuType, title } = props;
+  const { onClick, interfaceType, icon, isActive, menuType, title } = props;
+
+  const iconMap = {
```

Comment:
I'd be inclined to set an assumed format so that we don't need to keep adding to this list?

We already have change-case on the project so could do:

```
const icon = `menu-${changeCase.paramCase(interfaceType)}`;
```